content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/onboarding/mgm-onboarding.md

```diff
@@ -0,0 +1,572 @@
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+If using Bash, run the following, replacing `<holding-identity-ID>` with the ID returned in `holdingIdentity.shortHash` (for example, `58B6030FABDD`).
+```
+export MGM_HOLDING_ID=<holding-identity-ID>
+```
+
+## Assign Soft HSM and Generate Session Initiation and ECDH Key Pair
+
+To assign a soft high security module (HSM) and generate a session initiation key pair:
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```
+curl --insecure -u admin:admin -X POST $API_URL/hsm/soft/$MGM_HOLDING_ID/SESSION_INIT
+curl --insecure -u admin:admin -X POST $API_URL/keys/$MGM_HOLDING_ID/alias/$MGM_HOLDING_ID-session/category/SESSION_INIT/scheme/CORDA.ECDSA.SECP256R1
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Method Post -Uri "$API_URL/hsm/soft/$MGM_HOLDING_ID/SESSION_INIT"
+$SESSION_KEY_RESPONSE = Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Method Post -Uri "$API_URL/keys/$MGM_HOLDING_ID/alias/$MGM_HOLDING_ID-session/category/SESSION_INIT/scheme/CORDA.ECDSA.SECP256R1"
+$SESSION_KEY_ID = $SESSION_KEY_RESPONSE.id
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+If using Bash, the result contains `session key ID` (e.g. 3B9A266F96E2). Run the following command to save this ID for use in subsequent steps:
+```shell
+export SESSION_KEY_ID=<session key ID>
+```
+To generate an ECDH key pair:
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```
+curl --insecure -u admin:admin -X POST $API_URL/hsm/soft/$MGM_HOLDING_ID/PRE_AUTH
+curl --insecure -u admin:admin -X POST $API_URL/keys/$MGM_HOLDING_ID/alias/$MGM_HOLDING_ID-auth/category/PRE_AUTH/scheme/CORDA.ECDSA.SECP256R1
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Method Post -Uri "$API_URL/hsm/soft/$MGM_HOLDING_ID/PRE_AUTH"
+$ECDH_KEY_RESPONSE = Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Method Post -Uri "$API_URL/keys/$MGM_HOLDING_ID/alias/$MGM_HOLDING_ID-auth/category/PRE_AUTH/scheme/CORDA.ECDSA.SECP256R1"
+$ECDH_KEY_ID = $ECDH_KEY_RESPONSE.id
+```
+{{% /tab %}}
+{{< /tabs >}}
+If using Bash, the result contains `key ID` (e.g. 3B9A266F96E2). Run the following command to save this ID for use in subsequent steps:
+```
+export ECDH_KEY_ID=<ecdh key ID>
+```
+
+You can use the following schemes for ECDH key derivation:
+* ECDSA_SECP256R1
+* ECDSA_SECP256K1
+* X25519
+* SM2
+
+## Configure the Cluster TLS Key Pair and Certificate
+{{< note >}}
+This step is only necessary if setting up a new cluster.
+When using cluster-level TLS, it is only necessary to do this once per cluster.
+{{< /note >}}
+
+To set up the TLS key pair and certificate for the cluster:
+
+1. Create a TLS key pair at the P2P cluster-level by running this command:
```

Comment:
not sure if hyphen is needed here

Reply:
follow up to previous point....on line 550, it is used without hyphen
and then with a hyphen again on line 552